js/src/Ice/Logger.js

```diff
@@ -74,3 +76,27 @@ export class Logger {
         return d.toLocaleString("en-US", this._dateformat) + "." + d.getMilliseconds();
     }
 }
+
+let FileLogger = null;
+
+if (typeof fs.open === "function") {
+    FileLogger = class extends Logger {
+        constructor(prefix, filename) {
+            super(prefix);
+            this._filename = filename;
+        }
+
+        write(message, indent) {
```

Comment:
It would be nice to have a plain `write(message)` like in C#, and leave the formatting to the base class.